behaviour_and_protocol/consulting_101.md

```diff
@@ -1,26 +1,96 @@
+  * If you need time to eg. analyse a task or estimate changes, it is beneficial to communicate that to the client. Example: "Thank you for your message, I will take some time to analyse and get back to you by tomorrow."
+* **After a meeting, summarize key points and next steps in Slack or in Asana.**
+  * This ensures documentation and a shared memory on what has been decided.
+
+Technical Communication
+----------------------------------
+
+* **Tailor communication to the client's technical understanding.**
+  * Ask the client how technical they would like the explanation to be.
+  * If the client is non-technical, keep the explanations simple and eg. use metaphors.
+* **Provide realistic estimates and pose potential challenges and solutions.**
+  * Example: "We have estimated the SSO login as an XL-task. The documentation for the SSO is new to us, so we would have to spend some time researching. Alternatively, an email-based login would be an L-task."
+* **Explain system issues, highlighting the reasons for unexpected outcomes and the steps taken to fix these.**
+  * Example: "The issue occurred because the user could input text, and we were expecting numbers. We've updated the input and added additional tests."
+
+Etiquette and Professionalism
+-----------------------------
+
+* **Punctuality for meetings.**
+  * Some meetings require more preparation than others. Make sure to prepare for the meeting.
+  * Example: Arrive 5-10 minutes early to setup and review meeting materials.
+* **Stay professional in both physical and digital interactions.**
+  * You do not need formal attire for client meetings, but you should be presentable.
+  * Your messages are a part of your professional image. Make sure they are thought-through and polite.
+* **Ensure a professional setup for remote interactions.**
+  * Example: Ensure the background is tidy and well-lit for video meetings.
+* **Keep track of time spent on projects accurately and consistently.**
+  * Log your hours daily to maintain accurate project timelines and budgeting.
+
+Proactiveness
+-----------------------
+
+* **Be proactive in identifying and addressing potential issues or enhancements.**
+  * If you find something that could be fixed, could be optimized, or could be updated in other ways, communicate this to the team.
+  * Example: "I noticed a strange pattern in the data and wanted to make sure that this is not an error or would otherwise impact the project."
+* **Engage in proactive communication while being mindful of the implementation's impact.**
+  * Example: Before implementing a change, discuss potential impacts with the client and team.
+
+Solution-Oriented Approach
+--------------------------
+
+* **Avoid outright denials when addressing client requests; instead, propose alternative solutions or phased approaches.**
+  * Avoid: "That's impossible."
+  * Try: "That's challenging, but let's explore some alternatives that might work."
```

Comment:
```suggestion
* **Avoid outright denials when addressing client requests; instead, propose alternative solutions or phased approaches.**
  * Avoid: "That's impossible."
  * Try: "That's challenging, but let's explore some alternatives that might work."
  * Avoid: "It will take too much time."
  * Try: "This is a complex feature. If this is critical for the project, we can research and estimate it before we move on."
```